test/logoutrequest_test.rb

```diff
@@ -133,22 +121,24 @@ class RequestTest < Minitest::Test
     end
 
     describe "#create_params when the settings indicate to sign the logout request" do
-      def setup
-        @settings = OneLogin::RubySaml::Settings.new
-        @settings.compress_request = false
-        @settings.idp_sso_target_url = "http://example.com?field=value"
-        @settings.name_identifier_value = "f00f00"
-        @settings.security[:logout_requests_signed] = true
-        @settings.security[:embed_sign] = false
-        @settings.certificate  = ruby_saml_cert_text
-        @settings.private_key = ruby_saml_key_text
-        @cert = OpenSSL::X509::Certificate.new(ruby_saml_cert_text)
+
+      let(:settings3) { OneLogin::RubySaml::Settings.new }
```

Comment:
@pitbulk I can't disagree more ... descriptive names make the code **much** easier to read. There are better ways to deal with line length than making hard-to-understand variable names.